docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
             - 3000:3000
         depends_on:
             - postgres
+        volumes:
```

Comment:
*squints on the filename*

it should be moved to `docker-compose.formula.yml` to allow local usage of the repository without Salt (which would skip that file)

Reply:
The formula [uses `docker-compose -f docker-compose.yml -f docker-compose.formula.yml ...`](https://github.com/elifesciences/elife-xpub-formula/pull/32/files) commands to use both files, whereas locally by default `docker-composer ...` means `docker-compose [-f docker-compose.yml] ...`.

Hence everything that is specific to a formula-based server rather than being general (such as `/srv/elife-xpub/var/logs/`) needs to go into that file.